.github/workflows/component-bumper.yml

```diff
@@ -13,6 +13,11 @@ jobs:
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Pull latest master
+      run: git pull --rebase origin master
```

Comment:
I hate rebase commits. Would it make sense to go with --only-ff or do you see it bringing and complications?

Reply:
That's a great idea actually

Reply:
Another option is to do a range cherry-pick with all the commits on the PR.

Reply:
I think I prefer the `--only-ff` option. 
Done